README.md

```diff
@@ -158,3 +162,15 @@ test('something', async () => {
   expect(context.doSomething).toHaveBeenCalledTimes(3)
 })
 ```
+
+### Parallel Processing + Ordering
+
+By default, the abstractions in `@lifeomic/delta` (`DynamoStreamHandler` and `SQSMessageHandler`) will process events in parallel. To control the parallelization, specify a `concurrency` value when creating the handler.
+
+These abstractions also guarantee automatically ensure that within a batch of events correct _ordering_ is maintained according to the ordering semantics of the upstream event source, even when processing in parallel.
+
+In `DynamoStreamHandler`, events for the same _key_ will always be processed serially -- events from different keys will be processed in parallel.
+
+In `SQSMessageHandler`, events with the same `MessageGroupId` will always processed serially -- events with different `MessageGroupId` values will be processed in parallel.
+
+**Note**: when using a `concurrency` value of `1`, the ordering semantics above will still be preserved. But, events that do _not_ need to be ordered will not necessarily be processed in the same order they were received in the batch.
```

Comment:
Isn't this statement also true for any value passed to `concurrency`? (not only when it's `1`).